src/ModInterop.cs

```diff
@@ -0,0 +1,37 @@
+using Celeste.Mod.CommunalHelper.DashStates;
+using Monocle;
+using MonoMod.ModInterop;
+using System;
+
+namespace Celeste.Mod.CommunalHelper {
+
+    public static class ModExports {
+
+        internal static void Initialize() {
+            typeof(DashStates).ModInterop();
+        }
+
+        [ModExportName("CommunalHelper.DashStates")]
+        public static class DashStates {
+
+            #region DreamTunnel
+
+            public static int StDreamTunnelDash() => DreamTunnelDash.StDreamTunnelDash;
```

Comment:
As mentioned on Discord, a name like `GetDreamTunnelDashState` or `GetStDreamTunnelDash` would make a lot more sense for this. I personally prefer `GetDreamTunnelDashStateID`, I think it's as clear as you can get.